tests/database/hash.js

```diff
@@ -133,7 +134,8 @@ describe('Hash methods', function() {
 				assert.equal(arguments.length, 2);
 				assert.equal(typeof object, 'object');
 				assert.equal(object.lastname, 'usakli');
-				assert.equal(object.age, 99);
+				winston.info("it stores number as string in db! -- " + (typeof object.age));
+				assert.notStrictEqual(object.age, 99);
```

Comment:
Redis stores as string, Mongo as number. Best course of action is to use `equal`, not `strictEqual`. Or even better, use `strictEqual` and alternate `assert` statements based on db type.